tests/ElmJson.js

```diff
@@ -44,8 +44,16 @@ describe('handling invalid elm.json', () => {
           const elmJson = ElmJson.read(fullPath);
           ElmJson.requireElmTestPackage(fullPath, elmJson);
         },
-        {
-          message: expected,
+        (error) => {
+          assert.strictEqual(
+            error.message.replace(
+              // Handle slightly different JSON.parse error messages on different Node.js versions.
+              /^.+ in JSON at position .+$/gm,
+              '(the JSON parse error)'
+            ),
+            expected
+          );
+          return true;
```

Comment:
You can pass an assertion there 🤦‍♂️

Reply:
Yeah, I had to look that up the docs and play around with it in my editor. Otherwise I would have left a comment on your PR. How that’s ok 😄 

This turned out to be difficult to merge anyway due to other things being out of date, and me trying to ambitious not just do the simplest fix…

Reply:
That's fine 😁
I'd be the same way 😅